docs/examples/one_tank.md

```diff
@@ -0,0 +1,219 @@
+# One-Tank Example
+
+This example shows how to train two different models to predict the time-series behavior of a single water tank with a height-dependent outflow and a time-varying inflow.
+It is inspired by the one-tank simulink model used for control design [^1].
+A schematic drawing of the system is shown in the following figure.
+
+![The one-tank system](./images/one_tank.svg)
+
+The complete source code for this example can be found [in the repository](https://github.com/flowcean/flowcean/blob/main/examples/one_tank/run.py).
+See also [Run this example](#run-this-example) on how to run this example locally.
+
+The dynamics of the system can be described by an ordinary differential equation (ODE).
+This type of equation relates the change in a variable (it's derivative) to its actual value.
```

Comment:
I know what you mean, but this sentence may be confusing. Propostion: "ODEs describe the derivative of a variable as a function of the variable itself."